src/psp.rs

```diff
@@ -1414,15 +1414,19 @@ s_paren! {
 
     #[repr(transparent)]
     pub struct RegHandle(u32);
-
-    #[repr(C)]
-    pub struct sockaddr(pub u32);
-
-    #[repr(C)]
-    pub struct in_addr(pub u32);
 }
 
 s! {
+    pub struct sockaddr {
+        pub sa_len: u8,
+        pub sa_family: u8,
+        pub sa_data: [u8;14],
+    };
```

Comment:
This semicolon should be removed.

```suggestion
    }
```

Reply:
D'oh